### PR TITLE
PUBDEV-6500: Creating documentation for SVM 

### PR DESCRIPTION
**Pull request by zainhaq-h2o**
Creating documentation for SVM.
Edited list of hyperparameters (alphabetized, added SVM hyperparameters)
Added parameters to svm.rst

Do not merge.

https://0xdata.atlassian.net/browse/PUBDEV-6500

### CODE REVIEW

**Comment by michalkurka** [comment]
@zainhaq-h2o wow, that was quick and it looks good - please keep in mind that I am not yet done and the parameters will (likely) change

**Comment by michalkurka** [comment]
Also, we might need to rename SVM to something due to a naming collision in SW.

**Comment by zainhaq-h2o** [comment]
> Also, we might need to rename SVM to something due to a naming collision in SW.

Sounds good @michalkurka  - I'll change and / or add new parameters as needed.

File: h2o-docs/src/product/data-science/svm.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Interpreting an SVM Model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SVM Algorithm
+~~~~~~~~~~~~~
+
+References
+~~~~~~~~~~
+
+Implementation of ICF based on https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/34638.pdf
```
Comment (angela0xdata) [inline h2o-docs/src/product/data-science/svm.rst]: The references section should be a citation:
E.Y. Chang, K. Zhu , H. Wang, H. Bai, J. Li, Z. Qiu, H. Cui, Parallelizing support vector machines on distributed computers, in Proceedings of NIPS, 2007 <a href="http://papers.nips.cc/paper/3202-parallelizing-support-vector-machines-on-distributed-computers.pdf">Google Scholar</a>

You can see an example of the correct citation in the References section on this page:
https://link.springer.com/chapter/10.1007/978-3-642-20429-6_10

Comment (zainhaq-h2o) [reply h2o-docs/src/product/data-science/svm.rst]: Thanks for the info - adjusted that section in a new commit.

**Comment by AidanConnelly** [comment]
Is the SVM model going to be accessible from H2O flow?

**Comment by michalkurka** [comment]
@AidanConnelly eventually yes, but not with the first release

we usually don't roll out a new algo to Flow users, we make it available to R/Python users first - gather feedback, incorporate and then make available in Flow

**Comment by angela0xdata** [comment]
@zainhaq-h2o please indicate at the top that this is an alpha version of the documentation.

**Comment by michalkurka** [comment]
@zainhaq-h2o looks good to me - can we, for now, remove the missing section Algo description, Model interpretation and merge this?

**Pull request merged**